src/app/onion/learning-object-builder/components/content-upload/app/upload/upload.component.ts

```diff
@@ -455,24 +455,32 @@ export class UploadComponent implements OnInit, AfterViewInit, OnDestroy {
    * Handles file uploads by storing files in upload queue and uploading using the file manager
    *
    * @private
-   * @param {WebkitFile[]} files [List of files to enqueue and upload]
+   * @param {FileInput[]} files [List of files to enqueue and upload]
    * @memberof UploadComponent
    */
-  private handleUpload(files: WebkitFile[]) {
+  private handleUpload(files: FileInput[]) {
     this.enqueueFiles(files);
-    this.fileManager
-      .upload(this.bucketUploadPath, files)
-      .subscribe(update => this.handleUploadUpdates(update));
+    try {
+      this.fileManager
+        .upload(this.bucketUploadPath, files)
+        .subscribe(update => this.handleUploadUpdates(update));
+    } catch (e) {
+      if (e.name === UploadErrorReason.Credentials) {
+        this.handleCredentialsError();
+      } else {
+        this.error$.next('Unable to upload files at this time.');
```

Comment:
Is this a user-facing error? If so, we should clearly have a subject in the sentence to make sure that it is clear **we** are unable to upload files and the user should try again later.